apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistUtils.tsx

```diff
@@ -93,113 +95,134 @@ export type ChecklistStatus = {
 export const CHECKLIST_STATUSES: ChecklistStatus = {
   ServiceKey: {
     success: {
-      icon: getSuccessIcon('Service account is correctly installed'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Success!',
+      description: 'Service account key is valid',
       disabled: false,
     },
     invalid: {
       icon: getErrorIcon(
-        'Failed to connect to your service account. Re-install your service account key if the problem persists'
+        `We couldn't connect to Google Analytics APIs using the service account key provided. Try re-installing your previous service account key or create and install a new key`
       ),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Invalid service account and service account key',
+      description: 'Service account key is not valid',
       disabled: false,
     },
     missing: {
       icon: getErrorIcon(
-        'We were unable to retrieve the private key from your service account key file. Please try reinstalling your service account key file. If the problem persists, contact support.'
+        "For technical reasons we were unable to retrieve the service account key that's stored for this app. If the problem persists, try reinstalling the service account key or contact support."
       ),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Unable to retrieve service account key file.',
+      description:
+        'Unable to retrieve your stored service account key. Re-install it if the problem persists',
       disabled: false,
     },
   },
   AdminApi: {
     success: {
-      icon: getSuccessIcon('Admin API successfully enabled'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Success!',
+      description: 'Analytics Admin API is enabled',
```

Comment:
is `Analytics Admin API` a proper noun? it's weird to me to say `Analytics Admin API` vs `Google Analytics Admin API` or just `Analytics admin API` (non-proper form)

Reply:
I agree. Will fix in a few places where this is! (same for Data API)